test/e2e/create_test.go

```diff
@@ -47,6 +47,7 @@ func creationTestSuite(t *testing.T) {
 	t.Run("UserData validation", func(t *testing.T) { testUserData(t) })
 	t.Run("UserData idempotent check", func(t *testing.T) { testUserDataTamper(t) })
 	t.Run("Node Logs", func(t *testing.T) { testNodeLogs(t) })
+	t.Run("Metrics test", func(t *testing.T) { testMetrics(t) })
```

Comment:
"Metrics" can me multiple things here, e.g. it could also mean the prometheus side of things. Could we call this "Windows Exporter test", and also rename e2e test function and file?